src/components/ContactPage/ContactPage.js

```diff
@@ -1,5 +1,13 @@
+import React from "react";
+import NavigationBar from "../NavigationBar/NavigationBar";
+
 const ContactPage = () => {
-  return <div>ContactPage</div>;
+ return (
+    <div>
+      <NavigationBar />
```

Comment:
Everything looks cool and we can almost merge it. You still have to handle the navigation nicely. It's not good to add a navigation component to every view. Try to fix it. If you can't make it by tomorrow, talk to me.